python/lsst/daf/butler/core/storedFileInfo.py

```
@@ -244,3 +241,12 @@ def from_record(cls: Type[StoredFileInfo], record: Mapping[str, Any]) -> StoredF
             dataset_id=record["dataset_id"],
         )
         return info
+
+    def update(self, **kwargs: Any) -> StoredFileInfo:
```

Unrecognized keywords are just silently ignored?